packages/client-sdk-nodejs/package.json

```diff
@@ -19,7 +19,8 @@
     "integration-test-store": "jest storage.test.ts --maxWorkers 1",
     "unit-test": "jest unit",
     "integration-test-leaderboard": "jest leaderboard --maxWorkers 1",
-    "integration-test": "jest integration --testPathIgnorePatterns \"auth-client.test.ts|leaderboard.test.ts|storage.test.ts\" --maxWorkers 1",
+    "integration-test-control-cache-topics": "jest integration --testPathIgnorePatterns \"auth-client.test.ts|leaderboard.test.ts|storage.test.ts\" --maxWorkers 1",
+    "integration-test": "jest integration --maxWorkers 1",
```

Comment:
@momentohq/dev-eco lmk if anyone has any concerns about this change. It just seemed confusing / suprising to me that the default target didn't run all the tests.

Reply:
no concerns from my end 🚀